src/panels/lovelace/common/directives/action-handler-directive.ts

```diff
@@ -214,7 +214,7 @@ class ActionHandler extends HTMLElement implements ActionHandler {
     });
     element.addEventListener("click", element.actionHandler.end);
 
-    element.addEventListener("keyup", element.actionHandler.handleEnter);
+    element.addEventListener("keyup", element.actionHandler.handleKeyUp);
```

Comment:
👍, but the event should be `keyDown`, not `keyUp`.

Reply:
Ok, I will change that 🙂